moon/config/moon-round3.json

```diff
@@ -23,7 +23,8 @@
           "out": ["raw"],
           "init": {
             "mode": "PULL",
-            "endpoint": "tcp://localhost:5555"
+            "endpoint": "tcp://localhost:5555",
+	    "save_data": true
```

Comment:
do we want to keep saving ROS data? BTW maybe there is again some problem with tabs/spaces here

Reply:
I guess that's an emacs feature: "indentation commands insert (or remove) the shortest possible series of tab and space characters so as to align to the desired column". I should be able to configure it to use spaces only.

Reply:
File was using both 2 and 4 space indentation, will unify on 2 spaces

Reply:
also removed the no longer needed 'save_data' element